src/modules/addExp.js

```diff
@@ -1,13 +1,9 @@
-const checkEvent = async function (expAmount, response) {
+module.exports = amount => function addExp(response) {
     const { exp } = response;
 
-    response.exp = exp ? exp + expAmount : expAmount;
+    if (exp) {
```

Comment:
please dont use ternary operator =) if better for reading))

Reply:
It is arguable, imo ternary has fine readability